src/resources/restore_stage.sh

```diff
@@ -588,6 +583,6 @@ export DLPX_LIBRARY_SOURCE=""
 export REPLICATION_PASS=""
 export STAGINGPASS=""
 export SOURCEPASS=""
-env | sort  >>$DEBUG_LOG
+env | grep -v 'PASS' | sort >>$DEBUG_LOG
```

Comment:
Do we actually need this information in the log file. Can we remove this file entirely?